src/components/DocumentationTopic/Description/RequirementMetadata.vue

```diff
@@ -11,9 +11,9 @@
 <template functional>
   <!-- because the component is functional we must mannually pass the classes -->
   <p class="requirement-metadata" :class="data.staticClass">
-    <strong>Required.</strong>
+    <strong>{{ parent.$t('required') }}.</strong>
     <template v-if="props.defaultImplementationsCount">
-      Default implementation{{ props.defaultImplementationsCount > 1 ? 's' : '' }} provided.
+       {{ parent.$tc('metadata.default-implementation', props.defaultImplementationsCount) }}
```

Comment:
```suggestion
       {{ parent.$tc('metadata.default-implementation', props.defaultImplementationsCount > 1) }}
```
I think that's what you did somewhere else

Reply:
I think this version is right and the "somewhere else" place is maybe wrong. In the `ParameterAttributes` component, there is a place where it is doing this, and we likely want the actual numeric count and not a boolean (which the `> 1` would convert it to)

Reply:
` "default-implementation": "Default implementation provided. | Default implementations provided."` I think we are picking between these two options. And if it's > 1, we pick the later

Reply:
We need the second parameter to be a Number. In case it's plural, it will be the latter string.